podcasts/New Detail/PlaylistDetailViewController+EditActions.swift

```diff
@@ -0,0 +1,172 @@
+import UIKit
+import PocketCastsDataModel
+import PocketCastsServer
+import PocketCastsUtils
+
+extension PlaylistDetailViewController {
+    @objc func moreTapped() {
+        Analytics.track(.filterOptionsButtonTapped)
+
+        let optionsPicker = OptionsPicker(title: nil)
+
+        let chromecastAction = OptionAction(label: "Chromecast", icon: "nav_cast_off") {
+            Analytics.track(.filterOptionsModalOptionTapped, properties: ["option": "chromecast"])
+            self.castButtonTapped()
+        }
+        optionsPicker.addAction(action: chromecastAction)
+
+        let MultiSelectAction = OptionAction(label: L10n.selectEpisodes, icon: "option-multiselect") { [weak self] in
```

Comment:
nitpick: the variable name should start with lowercase:  `multiSelectAction`

Suggestion: could each option be created in a separated function or dynamic var? in order to make this method smaller?

Reply:
@SergioEstevao  I changed the name to be lowercase but this is the legacy code we have in the legacy filter. I didn't change anything here, just duplicated this part. Many parts are duplicated as long we will use the FF. I believe in general the way we handle the options should be refactored as the same code is duplicated in each VC